Gopkg.lock

```diff
@@ -338,7 +338,7 @@
   revision = "f0d7bb60956f88d4743f5fea66b5607b96d262c9"
 
 [[projects]]
-  digest = "1:3a07f8614c920cb5b898c9746186f63609d514e9aeeb738417d8c111feb76316"
```

Comment:
Is this update to `knative/pkg` intended?

Reply:
merged forward. this should be fixed